### DISCUSSION TITLE
How to install packages to add custom translation?

### DISCUSSION THREAD

**Discussion by shan52** [argosopentech/argos-translate#181] [Q&A]
How and where to install packages to add custom translation?

**Comment by PJ-Finlay** [comment]
You would create a package using the training scripts. Currently the only way to install packages is using the `argostranslate.package` Python module.

**Reply by PJ-Finlay** [reply]
This would be a good feature to add to argospm:

```
argospm install translate-en_es-1_5.argosmodel

```
